development/compiling/compiling_with_script_encryption_key.rst

```diff
@@ -1,10 +1,10 @@
 .. _doc_compiling_with_script_encryption_key:
 
-.. highlight:: shell
-
 Compiling with script encryption key
 ====================================
 
+.. highlight:: shell
```

Comment:
I did this in an attempt to get the page title to render correctly in the documentation changelog. Unfortunately, it still doesn't work:

![image](https://user-images.githubusercontent.com/180032/59747669-9fba5e00-9279-11e9-9298-b1255650c073.png)

Any pointers?

Reply:
That's weird, it should work. Try a build from scratch, maybe sphinx did not rebuild the index properly after this change?